packages/react-vapor/src/components/input/Input.tsx

```diff
@@ -195,6 +202,7 @@ export class Input extends React.Component<IInputProps, IInputComponentState> {
                 min={this.props.minimum}
                 max={this.props.maximum}
                 {..._.omit(this.props, [...PropsToOmitUtils.input, ...inputPropsToOmit])}
+                {...additionalProps}

```

Comment:
Does it work if we use directly `disabled={this.props.isReadOnly}` here?

Reply:
I cant because we already have the prop disabled so if I want to set disabled on the input in case of readOnly, I need to override the previous disabled set at line 204. This disabled is from the state so to avoid any problem with the input,  we need to set the additionalProps only when the props is set to true.